src/lesson 13/script.js

```diff
@@ -0,0 +1,260 @@
+const input = document.querySelector('#start_seconds');
+const btnTimeout = document.querySelector('#start_timeout');
+
+const userSeconds = () => {
+    alert('The End!');
+}
+
+btnTimeout.addEventListener('click', function() {
+    const timer = setTimeout(userSeconds, input.value * 1000);
+
+    setTimeout(() => { clearInterval(timer)}, (input + 1) * 1000);
+    input.value = '';
+});
+
+
+
+const date1 = new Date("7/11/2010");
+const date2 = new Date("12/12/2010");
+
+
+date.get
```

Comment:
I think there is something wrong, `date` is not defined, and what `get` should do?

<img width="625" alt="image" src="https://user-images.githubusercontent.com/84010501/199959035-2eb9c8c9-d124-4b03-92c1-fbe12aea41ae.png">